blockchain/ethereum/ethereumtest/chain.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository at
+// https://github.com/direct-state-transfer/dst-go
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ethereumtest
```

Comment:
It is named `ethereumtest` because there can be multiple test helper packages in the project and it is good to have the package name unique within the project.

Also,
- When a test uses helpers from two packages, say `ethereum/ethereumtest` and `node/nodetest`, no renaming of imported packages is required.
- `ethereumtest.NewChainBackendSetup` is more explicit in its meaning compared to `test.NewChainBackendSetup`. Because there will be more than one test packages, if it is named the other way.

Reply:
Also from the [tests in go-ethereum](https://github.com/ethereum/go-ethereum/blob/master/common/types_test.go#L104), `0x0000000000000000000000000000000000000000` is a valid address representation.